backend/tracim_backend/views/core_api/schemas.py

```diff
@@ -2139,6 +2139,12 @@ class RoleSchema(marshmallow.Schema):
     label = marshmallow.fields.String(required=True, example="Reader")
 
 
+class IdPConfigSchema(marshmallow.Schema):
```

Comment:
Rename `IdPConfigSchema` to `SAMLIdPConfigSchema` or `ShibboletIdpConfigSchema` or something clear that it is important for SAML/Shibb authentication only (and you dont care in "classical" case)

Reply:
done